### PR TITLE
Connect real data from API with map

### PR DESCRIPTION
**Pull request by TKS9988**
Connect real data from API with map

url : /map

### CODE REVIEW

**Comment by TKS9988** [comment]
Get current location information.
Displays nearby restaurants based on current location information.

The API is foursquare.

File: src/components/Map.jsx [hunk 1]
```diff
@@ -1,55 +1,68 @@
...
   const mapZoomLevel = 15;
 
-  const markers = DUMMY_COORDINATES.map((el) => {
-    const { lat, long } = el.coords;
-    const coords = [lat, long];
+  useEffect(() => {
+    if (!loading) {
+      const BASEURL = 'https://api.foursquare.com/v3/places/search?'
+      const parameters = {
+        client_id: import.meta.env.VITE_FORESQUARE_CLIENT_ID,
```
Comment (cherylli) [inline src/components/Map.jsx:18]: I think these are not required (Only required in the legacy api)
image

File: src/components/Map.jsx [hunk 2]
```diff
@@ -1,55 +1,68 @@
...
+        ll: coords,
+        open_now: 'true',
+        sort: 'DISTANCE'
+      }
+      axios
+        .get(BASEURL + new URLSearchParams(parameters), {
+          headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json;charset=UTF-8",
+            "Authorization": import.meta.env.VITE_FORESQUARE_API_KEY
```
Comment (cherylli) [inline src/components/Map.jsx:30]: should probably rename it to FOURSQUARE

**Review by cherylli — COMMENTED** [review]
works for me but I think the API call should probably be outside map as the results are required by multiple components. What does everyone else think?

**Pull request closed without merging**